active_directory/assets/configuration/spec.yaml

```
@@ -0,0 +1,15 @@
+name: Active Directory
+files:
+- name: active_directory.yaml
+  options:
+  - template: instances
+    options:
+    - template: instances/pdh
```

```suggestion
    - template: instances/pdh
       overrides:
           host.required: true
```
And then change default to false